src/components/display/fewlines/Form/Form.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styled from "styled-components";
+
+import { Button, ButtonVariant } from "../Button/Button";
+import { Input } from "../Input/Input";
+import { Form } from "./Form";
+
+export default { title: "Form", component: Form };
+
+export const StandardForm = (): JSX.Element => {
```

Comment:
Couldn't we make the same thing here as in the test? With a counter that shows that the submit will only be called once?

Reply:
That's actually a great idea!